<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>integralss</class>
 <widget class="QMainWindow" name="integralss">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>317</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
  background-color: rgb(72,209,204);
  color: white; 
  border: 1px solid CadetBlue;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_V">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>509</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(0,206,209);
  color: #0c6651; 
  border: 1px solid Teal;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0#20b2aa, stop: 1 #008b8b);
}</string>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ctan">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>387</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(0,206,209);
  color: #0c6651; 
  border: 1px solid Teal;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0#20b2aa, stop: 1 #008b8b);
}</string>
    </property>
    <property name="text">
     <string>ctan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>326</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid Teal;
color: #085c48
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0#20b2aa, stop: 1 #008b8b);
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_tan">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>326</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(0,206,209);
  color: #0c6651; 
  border: 1px solid Teal;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0#20b2aa, stop: 1 #008b8b);
}</string>
    </property>
    <property name="text">
     <string>tan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cos">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>265</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(0,206,209);
  color: #0c6651; 
  border: 1px solid Teal;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0#20b2aa, stop: 1 #008b8b);
}</string>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>387</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid Teal;
color: #085c48
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0#20b2aa, stop: 1 #008b8b);
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="intergralImg">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>38</y>
      <width>41</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;p&gt;&lt;img src=&quot;C:\Users\IT-baby\Desktop\calculate\imgint.png&quot; width=&quot;1000&quot; height=&quot;950&quot; alt=&quot;Винни-Пух&quot;&gt;&lt;/p&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="pixmap">
     <pixmap>../imgint.png</pixmap>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_acc">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>580</y>
      <width>110</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QRadioButton {

    color:                  teal;
}

QRadioButton::indicator {
    width:                  10px;
    height:                 10px;
    border-radius:          7px;
}

QRadioButton::indicator:checked {
    background-color:       teal;
    border:                 2px solid rgb(175,238,238);
}

QRadioButton::indicator:unchecked {
    background-color:       rgb(175,238,238);
    border:                 2px solid rgb(175,238,238);
}</string>
    </property>
    <property name="text">
     <string>accuracy:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Y">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>448</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(0,206,209);
  color: #0c6651; 
  border: 1px solid Teal;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0#20b2aa, stop: 1 #008b8b);
}</string>
    </property>
    <property name="text">
     <string>!</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="lower_limit">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>150</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit {
  background-color: rgb(175,238,238);
  color: Teal; 
  border: 1px solid DarkCyan;
}
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>128</x>
      <y>387</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid Teal;
color: #085c48
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0#20b2aa, stop: 1 #008b8b);
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_up">
    <property name="geometry">
     <rect>
      <x>126</x>
      <y>6</y>
      <width>100</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QRadioButton {

    color:                  teal;
}

QRadioButton::indicator {
    width:                  10px;
    height:                 10px;
    border-radius:          7px;
}

QRadioButton::indicator:checked {
    background-color:       teal;
    border:                 2px solid rgb(175,238,238);
}

QRadioButton::indicator:unchecked {
    background-color:       rgb(175,238,238);
    border:                 2px solid rgb(175,238,238);
}</string>
    </property>
    <property name="text">
     <string>upper_limit</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="function">
    <property name="geometry">
     <rect>
      <x>86</x>
      <y>80</y>
      <width>225</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit {
  background-color: rgb(175,238,238);
  color: Teal; 
  border: 1px solid DarkCyan;
}
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_decr">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>509</y>
      <width>183</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(32,178,170);
  color: Azure; 
  border: 1px solid Teal;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0#20b2aa, stop: 1 #008b8b);
}</string>
    </property>
    <property name="text">
     <string>&lt;&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>67</x>
      <y>265</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid Teal;
color: #085c48
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0#20b2aa, stop: 1 #008b8b);
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plus">
    <property name="geometry">
     <rect>
      <x>189</x>
      <y>387</y>
      <width>61</width>
      <height>122</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(0,206,209);
  color: #0c6651; 
  border: 1px solid Teal;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0#20b2aa, stop: 1 #008b8b);
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_min">
    <property name="geometry">
     <rect>
      <x>189</x>
      <y>326</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(0,206,209);
  color: #0c6651; 
  border: 1px solid Teal;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0#20b2aa, stop: 1 #008b8b);
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>128</x>
      <y>265</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid Teal;
color: #085c48
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0#20b2aa, stop: 1 #008b8b);
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ac">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>204</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(32,178,170);
  color: Azure; 
  border: 1px solid Teal;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0#20b2aa, stop: 1 #008b8b);
}</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_pro">
    <property name="geometry">
     <rect>
      <x>128</x>
      <y>204</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(0,206,209);
  color: #0c6651; 
  border: 1px solid Teal;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0#20b2aa, stop: 1 #008b8b);
}</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>128</x>
      <y>326</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid Teal;
color: #085c48
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0#20b2aa, stop: 1 #008b8b);
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_fun">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QRadioButton {

    color:                  teal;
}

QRadioButton::indicator {
    width:                  10px;
    height:                 10px;
    border-radius:          7px;
}

QRadioButton::indicator:checked {
    background-color:       teal;
    border:                 2px solid rgb(175,238,238);
}

QRadioButton::indicator:unchecked {
    background-color:       rgb(175,238,238);
    border:                 2px solid rgb(175,238,238);
}</string>
    </property>
    <property name="text">
     <string>function</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_x">
    <property name="geometry">
     <rect>
      <x>189</x>
      <y>509</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(0,206,209);
  color: #0c6651; 
  border: 1px solid Teal;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0#20b2aa, stop: 1 #008b8b);
}</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_lo">
    <property name="geometry">
     <rect>
      <x>91</x>
      <y>153</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QRadioButton {

    color:                  teal;
}

QRadioButton::indicator {
    width:                  10px;
    height:                 10px;
    border-radius:          7px;
}

QRadioButton::indicator:checked {
    background-color:       teal;
    border:                 2px solid rgb(175,238,238);
}

QRadioButton::indicator:unchecked {
    background-color:       rgb(175,238,238);
    border:                 2px solid rgb(175,238,238);
}</string>
    </property>
    <property name="text">
     <string>lower_limit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_10">
    <property name="geometry">
     <rect>
      <x>67</x>
      <y>204</y>
      <width>30</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(0,206,209);
  color: #0c6651; 
  border: 1px solid Teal;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0#20b2aa, stop: 1 #008b8b);
}</string>
    </property>
    <property name="text">
     <string>(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mul">
    <property name="geometry">
     <rect>
      <x>189</x>
      <y>265</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(0,206,209);
  color: #0c6651; 
  border: 1px solid Teal;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0#20b2aa, stop: 1 #008b8b);
}</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_dot">
    <property name="geometry">
     <rect>
      <x>128</x>
      <y>448</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid Teal;
color: #085c48
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0#20b2aa, stop: 1 #008b8b);
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>265</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid Teal;
color: #085c48
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0#20b2aa, stop: 1 #008b8b);
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sin">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>204</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(0,206,209);
  color: #0c6651; 
  border: 1px solid Teal;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0#20b2aa, stop: 1 #008b8b);
}</string>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_calc">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>615</y>
      <width>305</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(32,178,170);
  color: Azure; 
  border: 1px solid Teal;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0#20b2aa, stop: 1 #008b8b);
}</string>
    </property>
    <property name="text">
     <string>calculate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>448</y>
      <width>122</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid Teal;
color: #085c48
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0#20b2aa, stop: 1 #008b8b);
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="accuracy">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>578</y>
      <width>211</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit {
  background-color: rgb(175,238,238);
  color: Teal; 
  border: 1px solid DarkCyan;
}
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>67</x>
      <y>387</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid Teal;
color: #085c48
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0#20b2aa, stop: 1 #008b8b);
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="upper_limit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>6</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit {
  background-color: rgb(175,238,238);
  color: Teal; 
  border: 1px solid DarkCyan;
}
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>67</x>
      <y>326</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid Teal;
color: #085c48
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0#20b2aa, stop: 1 #008b8b);
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_11">
    <property name="geometry">
     <rect>
      <x>98</x>
      <y>204</y>
      <width>30</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(0,206,209);
  color: #0c6651; 
  border: 1px solid Teal;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0#20b2aa, stop: 1 #008b8b);
}</string>
    </property>
    <property name="text">
     <string>)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_dis">
    <property name="geometry">
     <rect>
      <x>189</x>
      <y>204</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(0,206,209);
  color: #0c6651; 
  border: 1px solid Teal;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0#20b2aa, stop: 1 #008b8b);
}</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>650</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
  background-color: rgb(175,238,238);
  color: Teal; 

}


background-color : white;</string>
    </property>
    <property name="text">
     <string>resoult:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Iresoult">
    <property name="geometry">
     <rect>
      <x>62</x>
      <y>650</y>
      <width>248</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
  background-color: rgb(175,238,238);
  color: Teal; 

}


background-color : white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
